src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -274,6 +278,22 @@ internal void WorkerInitResponse(GrpcEvent initEvent)
             _workerInitTask.SetResult(true);
         }
 
+        private void ParseAndLogWorkerMetadata()
+        {
+            if (_initMessage?.WorkerMetadata == null)
+            {
+                return;
+            }
+
+            _initMessage.WorkerMetadata.RuntimeName = string.IsNullOrEmpty(_initMessage.WorkerMetadata.RuntimeName)
+                                                        ? _workerConfig.Description.Language : _initMessage.WorkerMetadata.RuntimeName;
```

Comment:
nit: Could be simplified to 

````
_initMessage.WorkerMetadata.RuntimeName ??= _workerConfig.Description.Language;
````

Reply:
I actually started with that originally but it doesn't work in this case we `??=` only checks null and we need to check for null or empty. I ran debugged this with the dotnet worker and when `RuntimeName` was empty, it did not use the worker config language field

Reply:
You are right. My bad 😔